packages/attribute-registry-process/src/services/readModelService.ts

```diff
@@ -140,6 +140,41 @@ export const readModelService = {
     return undefined;
   },
 
+  async getAttributesByIds(
```

Comment:
I suggest to update [this](https://github.com/pagopa/interop-be-monorepo/pull/92/files#diff-cda50d168ca547dfedbc0861e72842888a70aac3daedeecb1e4a87f3f1f6c276R42) taking also the `ids`

Reply:
Refactored